tests/base/src/test/resources/org/wildfly/security/auth/client/wildfly-credential-sasl-config.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2017, Red Hat, Inc. and/or its affiliates, and individual
```

Comment:
This is a very minor comment but happened to notice 2017 here, I know this is just a copy/paste but we should use 2020 here :)